### PR TITLE
Changelog for 0.2.6.

### PR DESCRIPTION
**Pull request by abuiles**
Good to go, please review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+  + No changes required
+
+#### Community Contributions
+
+- [#4143](https://github.com/ember-cli/ember-cli/pull/4143) [BUGFIX] Blueprint.load verify blueprint is in a directory [@trabus](https://github.com/trabus)
+- [#4092](https://github.com/ember-cli/ember-cli/pull/4092) adding 0.2.5 diffs to changelog [@kellyselden](https://github.com/kellyselden)
+- [#4035](https://github.com/ember-cli/ember-cli/pull/4035) Add a verification step to fail the build when tests are filtered with .only [@marcioj](https://github.com/marcioj)
+- [#4091](https://github.com/ember-cli/ember-cli/pull/4091) fix name of ember-cli-dependency-checker [@bantic](https://github.com/bantic)
+- [#3854](https://github.com/ember-cli/ember-cli/pull/3854) [ENHANCEMENT] install:addon command will show a deprecation message before running the install command. [@DanielOchoa](https://github.com/DanielOchoa)
+- [#3550](https://github.com/ember-cli/ember-cli/pull/3550) Add ability to start ember serve on https [@drogus](https://github.com/drogus)
```
Comment (abuiles) [inline CHANGELOG.md]: user

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+- [#3786](https://github.com/ember-cli/ember-cli/pull/3786) Throw if templating a file fails [@davewasmer](https://github.com/davewasmer)
+- [#4026](https://github.com/ember-cli/ember-cli/pull/4026) Revert "Test powershell for appveyor builds" [@stefanpenner](https://github.com/stefanpenner)
+- [#4148](https://github.com/ember-cli/ember-cli/pull/4148) extract common SilentError debug/throw logic [@stefanpenner](https://github.com/stefanpenner)
+- [#4107](https://github.com/ember-cli/ember-cli/pull/4107) ENHANCEMENT More advanced detection of whether outputPath is a parent of the project directory [@catbieber](https://github.com/catbieber)
+- [#4104](https://github.com/ember-cli/ember-cli/pull/4104) [BUGFIX] Fix custom blueprint options for destroy command [@trabus](https://github.com/trabus)
+- [#4106](https://github.com/ember-cli/ember-cli/pull/4106) [ENHANCEMENT] Adding Report option to 'ember test' [@step2yeung](https://github.com/step2yeung)
```
Comment (abuiles) [inline CHANGELOG.md]: user

**Comment by stefanpenner** [comment]
LGTM

**Comment by stefanpenner** [comment]
hold-off

**Comment by stefanpenner** [comment]
#4187 is merged

**Pull request merged**